driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -1259,6 +1263,28 @@ synchronized void init() {
             try {
                 while (true) {
                     try {
+
+                        // create debouncers - at this stage, they are not running yet
+                        QueryOptions queryOptions = configuration.getQueryOptions();
+                        this.nodeListRefreshRequestDebouncer = new EventDebouncer<NodeListRefreshRequest>(
```

Comment:
+1.  I think it not entirely being accurate is ok as I think it will only mostly be used for tuning (what intervals to use and such) and for checking to make sure nothing is 'stuck'.